Fixes: Build error in develop

NishealJ opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
  - when this PR is merged.
  -->

## Checklist
- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The linter/Karma presubmit checks have passed.
  - These should run automatically, but if not, you can manually trigger them locally using `python scripts/pre_commit_linter.py` and `bash scripts/run_frontend_tests.sh`.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR has an appropriate "CHANGELOG: ..." label (If you are unsure of which label to add, ask the reviewers for guidance).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [x] The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).)
- [x] The PR is **assigned** to an appropriate reviewer.
  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer and don't tick this checkbox.
  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue. Do not only request the review but also add the reviewer as an assignee.

NishealJ committed:
Initial commit

NishealJ committed:
Change app.yaml to app_dev

kevinlee12 reviewed (approved):
lgtm!

apb7 commented on package-lock.json:
```diff
@@ -2184,18 +2187,615 @@
         "upath": "^1.1.1"
+      },
+      "dependencies": {
+        "fsevents": {
```
I don't think we should be checking in fsevents. It's unrelated to the fix and is required only on MacOS, right?

NishealJ replied:
Reverted Thanks @apb7

apb7 reviewed (commented):
Hi @NishealJ, thanks for the PR. Left a comment, ptal. Also, while you're pulling changes from different PRs, please do the changes made in PR #7136 so that we have a single PR for the fixes. Thanks!

kevinlee12 commented:
Somehow Github is not reporting the correct statuses: 

Fronted - PASS: https://circleci.com/gh/NishealJ/oppia/565
Setup - PASS: https://circleci.com/gh/NishealJ/oppia/564

NishealJ committed:
Revert Package lock

NishealJ commented:
> Somehow Github is not reporting the correct statuses:
> 
> Fronted - PASS: https://circleci.com/gh/NishealJ/oppia/565
> Setup - PASS: https://circleci.com/gh/NishealJ/oppia/564

Thanks, @kevinlee12, tests are now running in oppia's instance

NishealJ commented:
> Hi @NishealJ, thanks for the PR. Left a comment, ptal. Also, while you're pulling changes from different PRs, please do the changes made in PR #7136 so that we have a single PR for the fixes. Thanks!

Also @apb7, After the pull, I've removed the I/O try-catch block and updated only `app.yaml -> app_dev.yaml ` because the addition of exceptions was in discussion. Can we keep your open and the I/O expectation from that PR?

NishealJ commented:
Hi @DubeySandeep,
Sorry, I've pulled your PR and added it here, it was blocking me and was easy to debug the build-errors. Thanks for the fix!

apb7 commented:
> Also @apb7, After the pull, I've removed the I/O try-catch block and updated only app.yaml -> app_dev.yaml because the addition of exceptions was in discussion. Can we keep your open and the I/O expectation from that PR?

Please pull the complete PR changes (That PR is already LGTM). Thanks!

Commit:
Update third_party_size_check.py

apb7 reviewed (approved):
LGTM, thanks @NishealJ! Nice work :)

vojtechjelinek reviewed (approved):
LGTM!

DubeySandeep commented:
I'll have to cherry-pick this pr for the current release as after cherry-picking few PRs from develop the release branch started failing lint tests.
